osu.Game/Overlays/Notifications/NotificationSection.cs

```diff
@@ -75,7 +73,7 @@ private void load(OsuColour colours)
                     {
                         new ClearAllButton
                         {
-                            Text = clearButtonText,
+                            Text = NotificationsStrings.ClearAll,
```

Comment:
Was the removal of `ToUpper()` intentional?

Reply:
likely not